woocommerce/woocommerce-gutenberg-products-block
# [Store Customization MVP] Add fallbacks to all the pattern titles, descriptions and buttons

## What

This PR adds fallbacks to all the pattern titles, descriptions and buttons.

Part of https://github.com/woocommerce/woocommerce-blocks/issues/11166

## Why

To avoid a PHP warning that could be triggered if certain parts of the content for the pattern don't exist on the AI-generated content within the 'wc_blocks_patterns_content' option yet.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert all the patterns in the `dictionary.json` file.
3. Make sure they work as expected both on the editor side and on the front end.
4. Check your PHP error log and confirm you don't see any errors related to any pattern.


## Screenshots or screencast



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add a fallback to avoid the PHP warnings that could be triggered if the content saved within the "wc_blocks_patterns_content" option does not match the updated patterns dictionary.
